hnn_core/tests/test_cells_default.py

```diff
@@ -18,6 +19,17 @@ def test_cells_default():
     assert len(l5p.sections) == 9
     assert 'apical_2' in l5p.sections
 
+    # check that after building, the vertical sections have the length
+    # specified in get_L5Pyr_params_default (or overriden in a params file).
+    # Note that the lengths implied by _secs_L5Pyr are completely ignored:
+    # NEURON extends the sections as needed to match the sec.L 's
+    vertical_secs = ['basal_1', 'soma', 'apical_trunk', 'apical_1', 'apical_2',
+                     'apical_tuft']
+    for sec_name in vertical_secs:
+        sec = l5p.sections[sec_name]
+        vert_len = abs(sec.z3d(1) - sec.z3d(0))
+        assert allclose(vert_len, sec.L)
```

Comment:
this test passes on `master` ... in other words it's not testing the new functionality

this is why I suggest starting by writing the test first

Reply:
or I suppose the point of this PR is only to clarify the code?

Reply:
Indeed, this is intended to allow us to close #330 knowing that it won't bite us in the rear end when we start accepting section points other than those hard-coded in the private functions `_secs_*{Pyr|Basket}`